en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,321 @@
+# PrivateJet mode for Microgateways
+With many of applications gearing towards microservice architecture, it’s no surprise that container-orchestration systems such as Kubernetes have become so popular with functionalities such as automating computer application deployment, scaling, and management, 
+which simplifies a number of complex management tasks. WSO2 API-Manager provides cloud-native API management where user can expose microservices as managed APIs in cloud environments such as Kubernetes with the support of [WSO2-Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md). 
```

Comment:
WSO2 API Manager provides cloud-native API management, where a user is able to expose microservices as managed APIs in cloud environment such as Kubernetes. This could be done with the support of [WSO2-Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md)

Reply:
done